### ISSUE TITLE
About mIoU result on Cityscapes validation dataset

### ISSUE THREAD

**Issue by tanveer6715** [ydhongHIT/DDRNet#18]
Hi,

Thanks for your work and providing code. I am just confuse about the mIoU reported in the paper on cityscapes validation dataset which is around 77%. Kindly let me know is this mIoU achieved on pretrained model (imagenet)? I am training DDR-Net-Slim23 from scratch and I am getting around 55% mIoU on validation data of Cityscapes.

**Comment by tanveer6715** [comment]
Thank you. I guess you have also use validation dataset in training images that's may be the reason of high mIoU? I am just using 2975 training images of cityscapes dataset for training the model.

**Comment by ydhongHIT** [comment]
> Thank you. I guess you have also use validation dataset in training images that's may be the reason of high mIoU? I am just using 2975 training images of cityscapes dataset for training the model.

I didn't use validation dataset for training given that the network is evaluated on it. I use validation dataset for training to submit the results to the official server (evaluation on test set).
